### PR TITLE
Internationalise website

### LINKED ISSUE #2: Internationalise website
As a French-speaking user, I am presented with the following text on the homepage:

_title: "Transformer le code législatif en code logiciel"_

## Calculer les droits des citoyens

Calculez toute variable d’un système socio-fiscal à partir d’une situation individuelle.

## Prédire les impôts des entreprises

Modélisez tous types d’entités et évaluez leur éligibilité à des charges et à des allégements.

## Déterminer l’impact de réformes

Comparez la loi telle qu’elle est et telle qu’elle pourrait être.

- - - - -

- Redirect to the browser language if none is set in the URL: `openfisca.org/` redirects to `openfisca.org/fr/` if the browser preferred language is French.
- Serve the website in the language given by the first path element: `openfisca.org/fr/` is in French.

### PR DESCRIPTION
**Pull request by sandcha**
Fixes #2 

Use Hugo framework to translate openfisca.org from english to french.

### CODE REVIEW

File: layouts/partials/button-language.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{{ if .IsTranslated }}
+    {{ range .Translations }}
+        <a href="{{ .Permalink }}" class="cta lang">{{ .Site.Language.Lang }}</a>
```
Comment (Anna-Livia) [inline layouts/partials/button-language.html]: Could this be in the footer ?

File: config.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+contact = "Contact us"
+legal_notice = "Legal notice"
+
+
+[languages.fr]
+weight = 1
+pitch = "Transformer le code législatif en code logiciel"
+index = "Page d'accueil"
+countries = "Pays disponibles"
+analytics = "Analyses d'usage"
```
Comment (fpagnoux) [inline config.toml]: Statistiques d'usages ?

File: layouts/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<footer>
+    <ul>
+        <li><a href="{{ "index.html" | relLangURL }}">{{ $.Site.Params.index }}</a></li>
+        <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+        <li><a href="{{ "tracking" | relLangURL }}">{{ $.Site.Params.analytics }}</a></li>
+        <li>
+            <a id="github" target="_blank"><img src="/img/GitHub-Mark-Light-64px.png" alt=""/> {{ $.Site.Params.contribute }}</a>
```
Comment (fpagnoux) [inline layouts/partials/footer.html]: Theses links won't work anymore after this PR is merged: the point to `https://github.com/openfisca/openfisca.org/tree/master/public`, which will  be the same than `https://github.com/openfisca/openfisca.org/tree/hugo/public` afterwards. 

The script needs to be adapted.

File: content/countries.fr.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Country Packages disponibles"
+layout: countries
+
+definition: Les Country Packages sont les modules de base d'OpenFisca. Ils définissent les Paramètres, Entités et Variables d'un pays.
+introduction: "Les pays suivants ont leurs modélisations de systèmes socio-fiscaux :"
```
[the remark this answers is not in the document]
Comment (sandcha) [reply content/countries.fr.md]: Comme les modélisations sont majoritairement naissantes, plutôt pour `Il existe une modélisation des systèmes socio-fiscaux pour les pays suivants :`

File: content/_index.fr.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Transformer le code législatif en code logiciel"
```
[the remark this answers is not in the document]
Comment (sandcha) [reply content/_index.fr.md]: > Du code législatif au code logiciel ?

File: content/tracking.en.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Cookies and opt-out"
+layout: tracking
+
+introduction: "When you visit this website, we leave a small text file (a \"cookie\") on your computer. This helps us measure how many visits we\'ve had and which pages are most viewed."
```
[the remark this answers is not in the document]
Comment (sandcha) [reply content/tracking.en.md]: Not necessary for a protected sentence like this one.

**Comment by fpagnoux** [comment]
Ok, let's give up on browser language detection for now, and ship it as is 😞 .

It seems that Hugo was _really_ not thought to do this.

**Pull request merged**